content/millennium/r4/encounters/encounter.md

```diff
@@ -56,6 +56,7 @@ _Implementation Notes_
 
 * The [Encounter.hospitalization.destination] will be returned as a reference to a [contained] location resource.
 * The [Encounter.location.location] may be returned as a reference to a [contained] location resource.
+* A populated Encounter response bundle will be sorted based on the [Encounter.period].
```

Comment:
I think it may be specifically sorted on the start/beginning? If so, call that out.

Reply:
https://github.com/cerner/fhir.cerner.com/pull/381/commits/4d18a64cdcda5172c7387f7486acfbcf94b53713